src/Illuminate/Support/Pluralizer.php

```diff
@@ -136,14 +135,7 @@ public static function inflector()
         static $inflector;
 
         if (is_null($inflector)) {
-            $inflector = new Inflector(
-                new CachedWordInflector(new RulesetInflector(
-                    English\Rules::getSingularRuleset()
-                )),
-                new CachedWordInflector(new RulesetInflector(
-                    English\Rules::getPluralRuleset()
-                ))
-            );
+            $inflector = InflectorFactory::createForLanguage('english')->build();
```

Comment:
I think we should use their language constants.

```php
<?php

use Doctrine\Inflector\InflectorFactory;
use Doctrine\Inflector\Language;

$inflector = InflectorFactory::createForLanguage(Language::ENLISH)->build();

```

Reply:
My main goal actually was to make it easier to change the language ruleset, just like changing the locale in the `config/app.php` file, adding another configuration for the pluralizer ruleset. I was thinking something like:
```php
$inflector = InflectorFactory::createForLanguage(  $app['config']->get('app.pluralizer_ruleset', 'english')  )->build();
```
But I'm not sure how to fully implement it...
Other classes appear to get the config values from `$this->app`, however class Pluralizer does not extends any other class, so `$this` shoudn't have access to the app right?